src/vulkan-renderer/texture.cpp

```diff
@@ -1,27 +1,331 @@
 #include "inexor/vulkan-renderer/texture.hpp"
 
+// stb single-file public domain libraries for C/C++
```

Comment:
License comments (and the github url) is to much noise. This should be mentioned in the documentation.

Reply:
done.